src/_tailwind.css

```diff
@@ -0,0 +1,14 @@
+@import 'tailwindcss';
+
+@theme {
+  --color-primary: #00393f;
```

Comment:
These colors (primary to quinary) should all be set using the --mat-sys- equivalents.